pyproject.toml

```diff
@@ -24,6 +24,8 @@ django-redis = "==5.0.0"
 pandas = "*"
 openpyxl = "*"
 jsonschema = "^4.17.3"
+dateparser = "^1.1.7"
+xlrd = "^2.0.1"
```

Comment:
Why is this required?

Reply:
Well, for one of the xl files, pandas complained because xlrd was missing.

Reply:
xlrd is used as engine in pandas while reading old excel file..didn't new update in pandas helped?

Reply:
pandas is the latest. Maybe it was an issue with the file. Well, I guess we can get rid of xlrd for the moment and see if the real users get any issues related to xlrd.